README.md

```diff
@@ -30,11 +30,11 @@ Don't name arguments or flags with names that contain underscore symbols, becaus
 
 <!-- usage -->
 ```sh-session
-$ npm install -g @fluencelabs/cli
+$ npm install -location=global @fluencelabs/cli
```

Comment:
all this text is auto-generated so it won't be good to change it like that. But I still wanna know why you wanted to change this line. npm i -g something seems so to be the basic way of installing global npm dependencies. Doesn't it work for you? I asked guys with mac-s to check and I think it worked for them

Reply:
auto-generation is all good if it's correct. clearly
```bash
fluence (--version)
zsh: unknown file attribute: v```
```
is not correct and shouldn't be propagated.

```bash
npm -g install @fluencelabs/aqua
npm WARN config global `--global`, `--local` are deprecated. Use `--location=global` instead.
```
no point propagating deprecated ways

Reply:
cool. didn't see this warning ever before. If you don't wanna wait until they change it we should remove these tags and write this ourselves